config.go

```diff
@@ -219,6 +225,44 @@ func loadConfig() (*config, error) {
 		return nil, err
 	}
 
+	// Setup dial and DNS resolution functions depending on the specified
+	// options. The default is to use the standard golang "net" package
+	// functions. When Tor's proxy is specified, the dial function is set to
+	// the proxy specific dial function and the DNS resolution functions use
+	// Tor.
+	cfg.net = &torsvc.MultiNet{Tor: false}
+	if cfg.TorSocks != "" && cfg.TorDNS != "" {
+		// Validate Tor port number
+		torport, err := strconv.Atoi(cfg.TorSocks)
+		if err != nil || torport < 1024 || torport > 65535 {
+			str := "%s: The tor socks5 port must be between 1024 and 65535"
+			err := fmt.Errorf(str, funcName)
+			fmt.Fprintln(os.Stderr, err)
+			fmt.Fprintln(os.Stderr, usageMessage)
+			return nil, err
+		}
+
+		// If ExternalIPs is set, throw an error since we cannot
+		// listen for incoming connections via Tor's SOCKS5 proxy.
+		if len(cfg.ExternalIPs) != 0 {
```

Comment:
The attribute comment within the `Config` struct should be updated to indicate that it implements an outbound over Tor _only_ mode (meaning that listening will be disabled).